bindings/python/TypeTreeGeneratorAPI/TypeTreeGenerator.py

```diff
@@ -146,5 +151,20 @@ def get_nodes(self, assembly: str, fullname: str) -> List[TypeTreeNode]:
         DLL.FreeCoTaskMem(nodes_ptr)
         return nodes
 
+    def get_monobehavior_definitions(self) -> List[Tuple[str, str]]:
+        names_ptr = ctypes.POINTER(ctypes.c_char_p)()
+        names_cnt = ctypes.c_int()
+        assert not DLL.TypeTreeGenerator_getMonoBehaviorDefinitions(
+            self.ptr,
+            ctypes.byref(names_ptr),
+            ctypes.byref(names_cnt),
+        ), "failed to get module exports"
+        names_array = ctypes.cast(
+            names_ptr, ctypes.POINTER(ctypes.c_char_p * names_cnt.value)
+        ).contents
+        names = [name.decode("ascii") for name in names_array]
+        DLL.FreeCoTaskMem(names_ptr)
```

Comment:
This would leak memory, as the array pointers to the names wouldn't be freed.

Reply:
Which reminds me, that I still have to double check if the raw typetree struct leaks or not.

Reply:
I checked this with a debugger just now.
The problem is, that the iterator directly returns a bytes object and not a pointer.
So the managed free tries to free a native value reserved and used by python.
I'm quite surprised by this behavior as well.

Reply:
The issue is that the iterator doesn't work as intended.
I managed to fix it, but the solution isn't pretty.

So I'm simply going to include free function in the NativeAPI as well, which would also make usage in other languages next to python easier and less hacky.